tests/factories/doi.js

```diff
@@ -100,6 +101,18 @@ FactoryGuy.define('nameIdentifier', {
   },
 });
 
+FactoryGuy.define('relatedIdentifier', {
+  default: {
+    relatedIdentifier: '10.80225/rph240519sdfd',
+    relatedIdentifierType: 'DOI',
+    relationType: 'HasMetadata',
+    relatedMetadataScheme: 'DataCite',
+    schemeUri: 'https://schema.datacite.org/meta/kernel-4.3/doc/DataCite-MetadataKernel_v4.3.pdf',
```

Comment:
@kjgarza This is not a correct schemeUri.

Reply:
i'll can change in the next PR thanks